packages/palette/src/elements/Sup/Sup.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components"
+import { Text, TextProps } from "../Text"
+
+export type SupProps = TextProps
+
+export const Sup = styled(Text)`
```

Comment:
Doesn't actually need to be a `<sup>` tag since we don't actually need the semantics of superscript. Regarding the role (not tag) superscript, WAI-ARIA says:

> The superscript role is intended to be used only to mark up typographical conventions that have specific meanings; not for typographical presentation for presentation's sake. In general, authors SHOULD use this role only if the absence of the superscript would change the meaning of the content.